openaq_api/openaq_api/v3/routers/owners.py

```diff
@@ -41,10 +49,20 @@ def where(self) -> str:
         Returns:
             string of WHERE clause
         """
-        return "entities_id = :owners_id"
+        return "(owner->>'id')::int = :owners_id"
 
 
-class OwnersQueries(Paging):
+class OwnersQueries(
+    Paging,
+    RadiusQuery,
+    BboxQuery,
+    ProviderQuery,
+    OwnerQuery,
```

Comment:
this seems redundant. e.g. /v3/owners?owners_id=42 should just be in the path query /v3/owners/42

Reply:
Agreed.